ps_socialfollow.php

```diff
@@ -46,6 +44,11 @@ class Ps_Socialfollow extends Module implements WidgetInterface
         'instagram',
         'linkedin',
     ];
+    private $templateFile;
+    /**
+     * @var string
+     */
+    private $validation_message;
```

Comment:
`$templateFile` uses camelCase so maybe we should use camelCase for this variable too?